packages/core/theme/src/utilities/transition.ts

```diff
@@ -1,36 +1,36 @@
-const DEFAULT_TRANSITION_DURATION = "250ms";
+export const DEFAULT_TRANSITION_DURATION = "250ms";
 
 export default {
   /**
    * Transition utilities
    */
-  ".transition-all": {
-    "transition-property": "all",
-    "transition-timing-function": "ease",
-    "transition-duration": DEFAULT_TRANSITION_DURATION,
-  },
+  // ".transition-all": {
+  //   "transition-property": "all",
+  //   "transition-timing-function": "ease",
+  //   "transition-duration": DEFAULT_TRANSITION_DURATION,
+  // },
```

Comment:
do we actually need to keep them as comments? or we can remove them to keep the code clean?

Reply:
Okay, I also think it's unnecessary.